package.json

```diff
@@ -10,13 +10,7 @@
         "test:unit": "npm run delete:reports:unit && vitest --run --environment jsdom",
         "test:unit:watch": "npm run delete:reports:unit && vitest --environment jsdom",
         "test:e2e": "start-server-and-test dev 8080 'cypress open'",
-        "test:e2e:tablet": "start-server-and-test preview 8080 'cypress open --config viewportWidth=768,viewportHeight=1024'",
-        "test:e2e:desktop": "start-server-and-test preview 8080 'cypress open --config viewportWidth=1440,viewportHeight=900'",
```

Comment:
Why removing those ? I think they are good to keep for the moment and test them time to time.